### PR TITLE
Add config to build theme css bundles

### PR DESCRIPTION
**Pull request by dnewh**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -64,6 +64,7 @@
 
 # production
 /canopy/**/build
+/canopy**/dist
```
Comment (dplumb94) [inline .gitignore]: is this missing a '/'?

**Comment by dplumb94** [comment]
I would recommend adding a more descriptive description of the PR since the purpose is not obvious from the title.

File: .gitignore (hunk 1 above)
Comment (dnewh) [reply .gitignore]: nope

File: canopy/design-system/rollup.config.js [hunk 2]
```diff
@@ -0,0 +1,26 @@
+import sass from 'rollup-plugin-sass';
```
Comment (peterschwarz) [inline canopy/design-system/rollup.config.js]: Missing license header

File: canopy/design-system/src/themes/default/index.js [hunk 3]
```diff
@@ -0,0 +1 @@
+import './index.scss';
```
Comment (peterschwarz) [inline canopy/design-system/src/themes/default/index.js]: Ditto

File: canopy/design-system/rollup.config.js (hunk 2 above)
Comment (dnewh) [reply canopy/design-system/rollup.config.js]: added

File: canopy/design-system/src/themes/default/index.js (hunk 3 above)
Comment (dnewh) [reply canopy/design-system/src/themes/default/index.js]: added

**Pull request merged**